documentation/airport-guides/cyhz.md

```diff
@@ -0,0 +1,150 @@
+Opened in June 1960, it is now a hub for Air Canada and Air Canada Express.
+
+### Statistics
+3,908,799 passengers in 2015-2016
+80,278 aircraft movement in 2015-2016
+
+### Positions
+Not much is known about Halifax due to lack of local knowledge, however from [LiveATC feeds](https://www.liveatc.net/search/?icao=CYHZ), we know that Halifax Terminal on frequency 119.200MHz is used most.
+
+|Callsign             | Frequency  | Remarks
+|---------------------|:----------:|:---------------------------------------------:|
+|"HALIFAX DELIVERY"   | 123.950MHz |                                               |
+|"HALIFAX GROUND"     | 121.900MHz |                                               |
+|"HALIFAX TOWER"      | 118.400MHz |                                               |
+|                     |            |                                               |
+|"HALIFAX TERMINAL"   | 119.200MHz | Frequency used for both departure and arrival |
+|"HALIFAX TERMINAL"   | 118.700MHz | Looks to be seldom used                       |
+|"HALIFAX TERMINAL"   | 128.550MHz | Low altitude, looks to be rarely used         |
+|                     |            |                                               |
+|"HALIFAX ATIS"       | 118.025MHz |                                               |
+
+### About the Halifax Terminal Control Areas
+NAVCanada outlines the airspace boundary around Halifax in their Designated Airspace Handbook. All radar positions (departure, arrival) surrounding Halifax have the callsign "Halifax Terminal". Aircraft leaving the Halifax TCA should be handed over to Moncton Centre.
+
+[This diagram](http://prntscr.com/hv4bf4) may assist in visualising the airspace around Halifax.
+
+### Instrument Procedures
+Halifax has one Standard Instrument Departure and four Standard Terminal Arrival Routes.
+
+##### Departures
+All departing aircraft will be assigned the Halifax Four departure. The Halifax Four advises pilots to climb to 5000ft on runway heading. Upon receipt of an aircraft on departure, Halifax Terminal should: 
+* give a direct to a point along the filed route;
+* give vectors to a point along the filed route;
+* issue climb instructions.
+
+##### Arrivals
+Moncton Centre will assign STARs to aircraft and will give initial descent instructions.
+
+Halifax Terminal is responsible for the following:
+* issuing further descent clearances;
+* clearing aircraft onto a final approach, ILS, RNAV, Visual or LOC;
+* ensuring aircraft have sufficient spacing between them.
+
+Where possible, Terminal should issue descent clearances to meet the following restrictions:
+- Runway 05
+  * 5000' at PEPTA
+- Runway 14
+  * 5000' at VIMLA
+  * 5000' at ILOTA
+- Runway 23
+  * 5000' at RUDBU
+  * 5000' at PEPTA
+  * 5000' at LOGPO
+- Runway 32
+  * 5000' at MENVO
+  * 5000' at EBLUV
+  * 5000' at NOTUB
+
+The following fixes are Final Approach Course Fixes, meaning that aircraft can be cleared for an approach after these fixes:

```

Comment:
The following fixes are **Initial Approach** Fixes

Reply:
@Fechulo 
![image](https://user-images.githubusercontent.com/19408472/34565721-d5029942-f153-11e7-8c18-37cba10cbf70.png)

According to Jeppesen they're Final Approach Course Fixes: http://ww1.jeppesen.com/documents/aviation/business/ifr-paper-services/abb-glossary.pdf

Reply:
Gotta love chart inconsistencies...
![leros](https://user-images.githubusercontent.com/13125311/34565893-747ef0ec-f154-11e7-80b4-ad5e19df2c28.PNG)
Apparently `IF` stands for "intermediate fix", whatever that means. Just leave it then, not that important.

Reply:
@Fechulo The STAR charts say FACF and the ILS/RNAV charts say IF sooo I guess we can put both?

Reply:
If they use the RNAV approaches more, then it makes sense to call them FACF, otherwise IF would be better. The terminology doesn't matter anyway as long as the user knows what we're talking about.